.github/workflows/build.yml

```diff
@@ -0,0 +1,61 @@
+name: Build
+
+on: [push]
+
+jobs:
+  build:
+    name: Build for ${{ matrix.platform }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - kind: linux
+            os: ubuntu-latest
+            platform: linux
+          - kind: windows
+            os: windows-latest
+            platform: win
+          - kind: mac
+            os: macos-latest
+            platform: osx
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Node
+        uses: actions/setup-node@v3
+        with:
+          # The talk on the street says this might be a good version for building.
+          node-version: 14.20.1
+          cache: yarn
+
+      - name: Install Yarn dependencies
+        run: yarn install --frozen-lockfile
+
+      - if: matrix.platform == 'linux'
+        name: Install bsdtar # Required by electron-builder when targeting pacman.
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install -y libarchive-tools
+
+      - name: Build project
+        id: build
+        uses: StarUbiquitous/command-output@v1.0.1 # Store stdout/stderr to outputs.
+        # You might want to turn this on if there's problems with electron-builder.
+        # env:
+        #   DEBUG: electron-builder
```

Comment:
What do we gain by having this off?
Is it storage related? Faster builds?

Reply:
It was just quite verbose, outputting some 600 lines of debug info. I thought it might actually make it more difficult to spot the actual problem amidst all the noise.